bitnami/mariadb/values.schema.json

```diff
@@ -1,176 +1,1742 @@
+                        "selector": {
+                            "type": "object",
+                            "description": "Selector to match an existing Persistent Volume",
+                            "default": {}
+                        }
+                    }
+                },
+                "extraVolumes": {
+                    "type": "array",
+                    "description": "Optionally specify extra list of additional volumes to the MariaDB Primary pod(s)",
+                    "default": [],
+                    "items": {}
+                },
+                "extraVolumeMounts": {
+                    "type": "array",
+                    "description": "Optionally specify extra list of additional volumeMounts for the MariaDB Primary container(s)",
+                    "default": [],
+                    "items": {}
+                },
+                "initContainers": {
+                    "type": "array",
+                    "description": "Add additional init containers for the MariaDB Primary pod(s)",
+                    "default": [],
+                    "items": {}
+                },
+                "sidecars": {
+                    "type": "array",
+                    "description": "Add additional sidecar containers for the MariaDB Primary pod(s)",
+                    "default": [],
+                    "items": {}
+                },
+                "service": {
+                    "type": "object",
+                    "properties": {
+                        "type": {
+                            "type": "string",
+                            "description": "MariaDB Primary Kubernetes service type",
+                            "default": "ClusterIP"
+                        },
+                        "ports": {
+                            "type": "object",
+                            "properties": {
+                                "mysql": {
+                                    "type": "number",
+                                    "description": "MariaDB Primary Kubernetes service port for MariaDB",
+                                    "default": 3306
+                                },
+                                "metrics": {
+                                    "type": "number",
+                                    "description": "MariaDB Primary Kubernetes service port for metrics",
+                                    "default": 9104
+                                }
+                            }
+                        },
+                        "nodePorts": {
+                            "type": "object",
+                            "properties": {
+                                "mysql": {
+                                    "type": "string",
+                                    "description": "MariaDB Primary Kubernetes service node port",
+                                    "default": ""
+                                }
+                            }
```

Comment:
Be careful with schema changes in this case. The initial schema was generated initially for kubeapps and is not complete, if we add this new schema we can break some installations because is noto fully accurate, for example `nodePorts` should be a number.